src/components/Activity/index.js

```diff
@@ -0,0 +1,43 @@
+import { Card, Titulo, Info, Divisor } from "./styles"
+import PropTypes from "prop-types"
+
+/**
+ * Componente de Atividade do dia do evento
+ * @param {titulo, info} props "titulo" é o título da atividade
+ * que será realizada e "info" pode ser o nome do palestrante ou 
+ * alguma informação extra sobre a atividade.
+ * 
+ * Somente "titulo" é obrigatória, "info" é opcional.
+ * @returns 
+ */
+const Activity = (props) => {
```

Comment:
```suggestion
const Activity = ({ titulo, info}) => {
```
Com isso a gente n precisa sempre fazer "props.atributo"